xla/stream_executor/sycl/BUILD

```diff
@@ -0,0 +1,70 @@
+load(
```

Comment:
When I said that the line is duplicated, I only meant the `# buildifier: disable=out-of-order-load` line. (There were two of them.) Please add this back:
```
# Description:
#   SYCL-platform specific StreamExecutor support code.
```
(We can remove the buildifier tag if it isn't needed.)

Reply:
Got it. Done.